src/workerd/util/duration-alert.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include "kj/string.h"
+#include "kj/time.h"
```

Comment:
I think the preferred style is to use quotes for relative includes (usually just files in the same directory) and angle brackets for absolute includes.